components/BlogView.js

```diff
@@ -0,0 +1,51 @@
+import {useEffect} from 'react'
+import BlogPostItem from './navigation/BlogPostItem'
+import { getAllPosts } from '../lib/api'
+import ArrowRight from './icons/ArrowRight'
+import ArrowLeft from './icons/ArrowLeft'
+
+export default function BlogView() {
```

Comment:
Try to name that component in different way. `BlogView` sounds like a component of the whole site and not part of the site. `NewsroomBlogPosts` will be better.